Makefile

```diff
@@ -75,6 +75,9 @@ test: ## Spins up Docker resources and runs _every_ test
 	@docker-compose up -d test-runtime-deps
 	@cargo test --all --features docker -- --test-threads 4
 
+test-behavior: ## Runs behavioral tests
```

Comment:
Do we plan to add this to Circle? If so, it might be better to just add this to our Github actions workflow since we're starting to move that way.

Reply:
I’ve added it to the `CI` workflow.